detectors/node/opentelemetry-resource-detector-aws/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### Dependencies
+
+* The following workspace dependencies were updated
+  * devDependencies
+    * @opentelemetry/contrib-test-utils bumped from ^0.30.0 to ^0.31.0
```

Comment:
Maybe not for this PR, but I guess we don't really want these in the CHANGELOG, right?